docs/user/user/pings/custom.md

```diff
@@ -115,28 +119,6 @@ pings = load_pings(resource_filename(__name__, "pings.yaml"))
 
 </div>
 
-<div data-lang="C#" class="tab">
-
-In C#, this object must be registered with the Glean SDK from your startup code (such as in your application's `Main` method or a function called from that method).
-
-```C#
-using static Mozilla.YourApplication.GleanMetrics.Pings;
-
-...
-
-class Program
-{
-    static void Main(string[] args)
-    {
-        ...
-        Glean.RegisterPings(Pings);
-        ...
-    }
-}
-```
-
-</div>
-
 {{#include ../../../shared/tab_footer.md}}
 
 ## Sending metrics in a custom ping
```

Comment:
This section is reference for the Ping.submit API. It should be moved to the API Referece section under a "Ping" page. In that page is where we would also document the `testBeforeNextSubmit` API.